### PR TITLE
Fix remove missing content

### PR DESCRIPTION
**Pull request by knod**
The cleanup from last night.

### CODE REVIEW

File: docassemble/juvenilesealing/data/questions/petitioner-entrypoint-10.yml [hunk 1]
```diff
@@ -164,7 +164,9 @@ continue button label: Next ${next_icon}
 
 under: |
   Read more about sealing juvenile records in the
-  [GBLS 2019 booklet ${external_link_icon}](https://www.gbls.org/sites/default/files/2019-08/juvenile_booklet_august_2019F.pdf).
+  [GBLS 2019 booklet ${ external_link_icon }](https://www.gbls.org/sites/default/files/2019-08/juvenile_booklet_august_2019F.pdf).
+  
+  <span class="feedback">Let us know <a href="blah">how we're doing ${ external_link_icon }</a>.</span>
```
Comment (niharikasingh) [inline docassemble/juvenilesealing/data/questions/petitioner-entrypoint-10.yml:169]: Do we have a link that should go here, or should this just stay `href="blah"` for now?

Comment (niharikasingh) [reply docassemble/juvenilesealing/data/questions/petitioner-entrypoint-10.yml:169]: Also, if we're putting the feedback link into the app, I think we also need to add the link in one more place (the shared file, so the link will show up on all the pages, not just the first page).

File: setup.py [hunk 2]
```diff
@@ -45,7 +45,7 @@ def find_package_data(where='.', package='', exclude=standard_exclude, exclude_d
...
...estions\r\n1. Downloadable/printable pre-filled petition PDF\r\n1. Provide the address for mailing the petition\r\n\r\n## Users\r\n\r\n### Target Audiences\r\n\r\n1. Petitioners.\r\n1. Legal professionals or clinic volunteers.\r\n\r\n### Unresovled Petitioner Barriers\r\n\r\n1. Frustration and distrust of the process.\r\n1. No access to printers.\r\n1. Which vocabulary will they find familiar?\r\n\r\n**User research needs to play a large role in the design of this tool.**\r\n\r\nThis tool is made with [Docassemble](https://github.com/jhpyle/docassemble).\r\n\r\n## Testing\r\n\r\n### Chai/Puppeteer tests\r\n\r\nThe first time you clone this repo, make sure you have NPM installed and run `npm install`.\r\nThis will install the libraries required for testing.\r\n\r\nTo run the tests, run `npm run test`.\r\n\r\n### Aloe/Lettuce tests\r\n\r\nTo get these tests to run, you need to download chromedriver from https://sites.google.com/a/chromium.org/chromedriver/ and put it in `tests/`\r\nYou also need to `pip3 install aloe selenium certifi`\r\n\r\nTo run [a feature test](https://docassemble.org/docs/development.html#bdd):\r\n```\r\ncd tests\r\naloe features/hello_world.feature\r\n```\r\n',
```
Comment (knod) [inline setup.py:48]: Anyone know why this file got changed? I don't remember editing it for this PR. Or anytime recently.

Comment (knod) [reply setup.py:48]: This should have been pulled in with the previous PR, I think.

Comment (niharikasingh) [reply setup.py:48]: I think maybe the readme changes didn't get pulled on the last PR because I never went into playground?

**Pull request merged**